### ISSUE TITLE
eslint-plugin-tsdoc

### ISSUE THREAD

**Issue by pizzafox** [xojs/eslint-config-xo-typescript#30]
Should this configuration enforce Microsoft's [`eslint-plugin-tsdoc` ](https://www.npmjs.com/package/eslint-plugin-tsdoc)plugin?
> This ESLint plugin provides a rule for validating that TypeScript doc comments conform to the [TSDoc specification](https://github.com/microsoft/tsdoc).

**Comment by sindresorhus** [comment]
Yes, eventually, but I don't think it makes sense until the TSDoc spec is stable. Right now it's just a draft.

**Comment by fregante** [comment]
tsdoc is not configurable. I'd suggest just using the jsdoc plugin and set `types: false` in the config: https://www.npmjs.com/package/eslint-plugin-jsdoc
